### ISSUE TITLE
[bug]: On Freepbx 17 script, Apache doesnt auto redirect you to /admin, end user needs to put /admin to bring up GUI

### ISSUE THREAD

**Issue by jlizzotte** [FreePBX/issue-tracker#34]
### FreePBX Version

FreePBX 17

### Issue Description

On Freepbx 17 script, Apache doesnt auto redirect you to /admin, end user needs to put /admin to bring up GUI. Also, after install and setup, I had to run:
a2enmod rewrite
systemctl restart apache2
Otherwise, It would say page cannot be displayed

### Operating Environment

Vultr installed of Debian 12

**Comment by thadrumr** [comment]
It looks like the default apache webpage is not change to /admin/config.php. The Default Apache webpage is still installed after the script is finished.

**Comment by kapilgupta01** [comment]
Hi @thadrumr did you tried using below steps - 
https://github.com/FreePBX/sng_freepbx_debian_install/commit/e7df5ce3f26cee67083aefccc515804301765240 

May be if you have tried "curl https://raw.githubusercontent.com/FreePBX/sng_freepbx_debian_install/master/sng_freepbx_debian_install.sh | bash" then it may have timed out due to which script might not have executed properly.
you can check "/var/log/pbx/freepbx17-install.log" log to confirm if your last lines clearly says execution finished successfully or not.

**Comment by tony1661** [comment]
I had the same issue. There may be some issues with the install script. I tried 3 times to install it on fresh VMs and only on the 3rd attempt did everything go smoothly.

When the script finished, did you get the FreePBX splash screen?

**Comment by jlizzotte** [comment]
Only after manually adding /admin did i get the splash screen

**Comment by kapilgupta01** [comment]
issue could be due to the curl timeout, so if you try steps as i mentioned in previous comment then also do you see timeout or exit in middle? 
yes at the end of the script we are executing "fwconsole motd" to show the freepbx banner.

**Comment by thadrumr** [comment]
I followed the instructions in the blog post linked here to perform my install.  https://www.freepbx.org/freepbx-17-official-debian-install-beta/

More specifially I gave the script execute permissions using chmod +x sng_freepbx_debian_install.sh
Then I executed the script ./sng_freepbx_debian_install.sh

Also your commit has the wrong link.  You should be using the raw link like below.

https://github.com/FreePBX/sng_freepbx_debian_install/raw/master/sng_freepbx_debian_install.sh

**Comment by kapilgupta01** [comment]
yes so @thadrumr that could be the reason, curl might have timed out and left the process in middle, check your log file to see if its finished properly or not. If not then re-run the script.

thanks for the wrong url notice, updated readme.

thanks

**Comment by kapilgupta01** [comment]
Please try with ["how-to-execute-the-script"](https://github.com/FreePBX/sng_freepbx_debian_install/blob/master/README.md#how-to-execute-the-script)  steps and re-open this jira if issue still exist.

Thank you for trying FreePBX 17
Kapil Gupta

**Comment by karmat0se** [comment]
I am seeing this issue as well. I did a fresh install on Debian 12.4 this morning. I executed the install script as instructed in the readme.md and am using the currently available shell script to install things.  

The log file states the following:

`2024-03-20 10:20:23 - Installation successful.

2024-03-20 10:20:23 - Total script Execution Time: 831`

.... However, it appears the default apache webpage is still not changing to /admin/config.php. Re-running the install script a couple times did resolve the problem though.
